packages/calcite-components/src/utils/text.tsx

```diff
@@ -4,6 +4,16 @@ const CSS = {
   textMatch: "text-match",
 };
 
+/**
+ * Capitalizes the first letter in a provided word.
+ * Assumes an unbroken string of text representing a single word.
+ *
+ * @param word string
+ */
+export function capitalizeWord(word: string): string {
```

Comment:
@eriklharper could this be done with CSS? https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform